platform/android/java_godot_view_wrapper.cpp

```diff
@@ -64,6 +65,11 @@ void GodotJavaViewWrapper::release_pointer_capture() {
 	}
 }
 
+void GodotJavaViewWrapper::set_pointer_icon(int pointer_type) {
+	JNIEnv *env = get_jni_env();
+	env->CallVoidMethod(_godot_view, _set_pointer_icon, pointer_type);
```

Comment:
Follow the pattern in the preceding methods:
1. Check that `_set_pointer_icon` is valid.
2. Check that `env` is not `null`.

Reply:
fixed 😄